activity/doc.go

```diff
@@ -103,9 +103,9 @@ It is also possible to heartbeat an activity from an external source:
     client.Client client = client.NewClient(...)
 
     // record heartbeat
-    err := client.RecordActivityHeartbeat(taskToken, details)
+    err := client.RecordHeartbeat(taskToken, details)
 
-The parameters of the RecordActivityHeartbeat function are:
+The parameters of the RecordHeartbeat function are:
```

Comment:
of the activity.RecordHeartbeat and client.RecordActivityHeartbeat functions

Reply:
I've rewritten the parapgraph since activity.RecordHeartbeat has a different signature (without the taskToken parameter).